### DISCUSSION TITLE
Managing End user preferences

### DISCUSSION THREAD

**Discussion by beckersb-dot** [iterate-ch/cyberduck#16117] [Q&A]
Hi,

I looked a bit in the discussion but did not find the answer to my question... Maybe I did not search properly :)

Is it possible to manage end user preferences to lockdown some settings ?

Thanks a lot

**Comment by tatze96** [comment]
The available setting can be adjusted or hidden with the help of hidden properties. However, we don't have a public list of them available, as as most of them are mainly meant for solving temporary issues.

Which settings do you want to adjust or lockdown?
